jni_gen/third_party/ApiSummarizer/src/main/java/com/github/hegde/mahesh/apisummarizer/elements/JavaDocComment.java

```diff
@@ -0,0 +1,14 @@
+package com.github.hegde.mahesh.apisummarizer.elements;
+
+public class JavaDocComment {
+  public String comment;
+
+  // TODO: Escape HTML tags, and optionally convert the familiar ones to markdown.
+
+  // TODO: Build a detailed tree representation of JavaDocComment
```

Comment:
File a bug on this repo, and link to the bug. (Idem other TODOs. Ctrl+f TODO and see if all have a link to a GitHub issue. 😄)

Reasoning for TODOs on GitHub instead of in code: it is easier to discuss alternative solutions and if we want to prioritize it on a GitHub issue. Also a GitHub issue has more space for elaborating (instead of leaving a 10 line TODO in the code).